### ISSUE TITLE
public tags vs private tags

### ISSUE THREAD

**Issue by amyxzhang** [haystack/eyebrowse-server#75]
Something that is a bit up in the air in my mind.

Right now we have tags implemented as private, at the domain level, and only 1 per domain. They were built in mind for quantified self people trying to make sense of their own history (although you can currently see these tags in the following and firehose feeds too...) Some enhancements to this (allow more than 1 for a domain for ex) could be done and makes sense in the personal context.

I'd like to add the idea of a public tag. These would be visualized as #tag-name at the bottom of an eyehistory item in any feed. Anyone can add a tag by commenting with #tag-name while on the page. You can have as many as you want per page. This gets us closer to allowing people to curate the content element of their feeds by explicitly muting public tags and searching by tag (like twitter!)

I'd like to bootstrap this with delicious tags. I'm worried about hitting delicious's rate limit, but David had the idea of getting tags via the chrome extension code. This solves the problem of ip-address rate limiting but not account (we have an Eyebrowse delicious account for this).

I've also created the ability to sync one's delicious acct but so far this does nothing. I'm of a mind to delete this functionality as opposed to build it out as I don't want to further complicate the tags people see, and i think most ppl no longer use delicious anyway.

I'd like to think of terminology to separate the two types of tags. Let's call the private ones something like "My Labels" and the public ones Hashtags?

**Comment by amyxzhang** [comment]
still thinking about how to extract tags. we can also use meta keywords and meta opengraph article:tag. I'm going to try with that first and see what the quality is like. 

also going to work on allowing user-generated tags via leaving a hashtag in a note.

**Comment by karger** [comment]
i don't remember exactly what kind of tag extraction we are undertaking 
here.

**Comment by amyxzhang** [comment]
interested in topical keywords

**Comment by karger** [comment]
On 3/24/2015 6:18 PM, Amy Zhang wrote:
>
> interested in topical keywords
>
this is a nontrivial task to solve well; will it be beneficial to do it 
not-well or do you just want to rely on manually authored tags + delicious?

**Comment by amyxzhang** [comment]
yes it's not easy. I was thinking a combo of delicious + meta keyword + meta opengraph article:tag for bootstrapping. I think with all these three, we'll have decent coverage....

or maybe i'll just focus on enabling manual generation for now.

**Comment by karger** [comment]
On 3/24/2015 6:22 PM, Amy Zhang wrote:
>
> yes it's not easy. I was thinking a combo of delicious + meta keyword 
> + meta opengraph article:tag for bootstrapping. I think with all these 
> three, we'll have decent coverage....
>
i don't think of any of those as "extraction" because there's no ML 
involved.  so they seem safe.
